### DISCUSSION TITLE
Feature Idea: Crossplane + Kube Green

### DISCUSSION THREAD

**Discussion by dr460neye** [kube-green/kube-green#370] [Ideas]
Crossplane is orchestrating full cluster stacks. 
It would be a charm if you consider adding crossplane support to not get cross product reactions and improve kube-green at big scale.
The impact could be massive.

**Comment by davidebianchi** [comment]
Hi @dr460neye, I totally agree with you! There are already some suggestions about this topic in #367.

It's something quite new, but with the work in #363 it could be achieved! After that, we should try it with crossplane and see if something other is needed.
